### ISSUE TITLE
 Import "google/protobuf/descriptor.proto" was not found or had errors 

### ISSUE THREAD

**Issue by aantonop** [lightninglabs/lightning-terminal#152]
Trying to compile LiT from source on Ubuntu 18.04LTS, with go 14.1 and node 12.19, yarn 1.22

```
yarn run v1.22.10
$ node ./scripts/build-protos.js
Compiling protobuf definitions
protoc --plugin=protoc-gen-ts=/usr/local/gocode/src/github.com/lightninglabs/lightning-terminal/app/node_modules/.bin/protoc-gen-ts --proto_path=../proto --js_out=import_style=commonjs,binary:./src/types/generated --ts_out=service=grpc-web:./src/types/generated ../proto/lnd.proto ../proto/loop.proto ../proto/google/api/annotations.proto ../proto/google/api/http.proto
Error: Command failed: protoc --plugin=protoc-gen-ts=/usr/local/gocode/src/github.com/lightninglabs/lightning-terminal/app/node_modules/.bin/protoc-gen-ts --proto_path=../proto --js_out=import_style=commonjs,binary:./src/types/generated --ts_out=service=grpc-web:./src/types/generated ../proto/lnd.proto ../proto/loop.proto ../proto/google/api/annotations.proto ../proto/google/api/http.proto
google/protobuf/descriptor.proto: File not found.
google/api/annotations.proto: Import "google/protobuf/descriptor.proto" was not found or had errors.
google/api/annotations.proto:28:8: "google.protobuf.MethodOptions" is not defined.
loop.proto: Import "google/api/annotations.proto" was not found or had errors.
```

**Comment by guggero** [comment]
Thank you for reporting. We definitely have to make it more explicit, but I think you are missing a library.
Can you try if installing the protobuf depencencies [as described here](https://github.com/lightningnetwork/lnd/tree/master/lnrpc#generate-protobuf-definitions) fixes the problem?
I'll create a PR to mention this in the docs if that indeed is what's missing.

**Comment by jamaljsr** [comment]
Yes, this error is caused by `protoc` not being properly installed. The `google/protobuf/descriptor.proto` file is included in the `protoc-3.14.0-linux-x86_64.zip` archive. You only need to perform step #1 in the instructions that Oli linked to. There's also `protoc` installation instructions on the [gRPC](https://grpc.io/docs/protoc-installation/) website which is not much different. I've installed on Ubuntu multiple times using these commands. You can try both to see which works for you. I think it ultimately just depends on setting your `PATH` correctly.

**Comment by aantonop** [comment]
Thank you! This would be useful, even if only as a link, in the README
